src/style.css

```diff
@@ -1 +1,54 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
```

Comment:
add horizontal padding 50px to header, remove margins to make distance from browser window

Reply:
use justify-content: space-between to create distance between logo and nav

Reply:
center logo and nav with flexbox (add this to header)